Example/Example/AppDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AppDelegate.swift
```

Comment:
Why is this empty Example project added?

Reply:
@toupper What do you mean empty? We had `PiedPiperSample` before with some exaple code inside the `ViewController.swift`. The new `Example` project also has the same code inside `ViewController.swift`. The only difference is the new Example project structure and compatibility with the latest Xcode

Reply:
Ah cool, I missed that. Maybe we can remove the template functions we don't use, such as `func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {`?

Reply:
👍. Make sense, I will clean up the example project

Reply:
@toupper I've cleaned up the Example project